### PR TITLE
Added KEYS.ENTER to submit login data

### PR DESCRIPTION
**Pull request by CharlesCCC**
Currently running into issue when login into account. looks like instagram is changed their xpath again for login button.

### CODE REVIEW

**Comment by timgrossmann** [comment]
@CharlesCCC Awesome, thank you very much for this update! 
Could you please make sure that the CI passes? :blush:

**Comment by CharlesCCC** [comment]
@timgrossmann  I would love to, but TBH, I have no idea how to fix that, As I view the log,it failed to call ` tox` command ??? Is there anyway I can kick off the build again ?

File: instapy/login_util.py [hunk 1]
```diff
@@ -259,9 +259,23 @@ def login_user(browser,
         login_button = browser.find_element_by_xpath(
             "//div[text()='Log in']")
```
Comment (converge) [inline instapy/login_util.py]: I believe the "Log in" way doesn't exist, am I wrong?

I'm just able to find "Log In" button.

File: .gitignore [hunk 2]
```diff
@@ -104,6 +104,7 @@ logs/
 # Editors
 .idea
 .vscode
+.bak
```
Comment (converge) [inline .gitignore:107]: In this for Mysql Workbench backup file?

**Review by converge — CHANGES_REQUESTED** [review]

File: instapy/login_util.py (hunk 1 above)
Comment (CharlesCCC) [reply instapy/login_util.py]: You probably right, I was accumulated all the possible option from the previous changes.

File: .gitignore (hunk 2 above)
Comment (CharlesCCC) [reply .gitignore:107]: actually, it was from Notepad++ file.  but it could be others as well.

Comment (hugio55) [reply .gitignore]: I am pretty new to this whole world. Where is this .gitignore file?

**Comment by timgrossmann** [comment]
@CharlesCCC @converge which changes do you exactly want for this PR?
I think it's good to cover "every" possible combination here.

Please define so we can merge this 😊

**Comment by CharlesCCC** [comment]
@timgrossmann I would say keep all the possible. There should be nothing to hurt. worst case is it take couple more second to login the user, but (it just one second every 3 months, since the cookie are cached).

**Comment by converge** [comment]
It's all good for me, I just don't know how to deal with the "Log In" vs "Log in". It's hard to test it because it just catches "Log In" on my side, and I don't know if people catch "Log in".

**Comment by mkhattar** [comment]
I changed "Log in" to "Log In", seems the text on the button changed. Please can someone else test it to confirm?

**Comment by CharlesCCC** [comment]
@mkhattar @converge I have replaced "log in" with "Log In".

**Review by converge — APPROVED** [review]

**Comment by orhan-drsn** [comment]
What about:
`from selenium.webdriver.common.keys import Keys`

`(ActionChains(browser)`
`        .move_to_element(input_password[0])`
        `.click()`
        `.send_keys(Keys.ENTER)`
        `.perform())`

Maybe this is the best way to don't have problems in future

**Comment by CharlesCCC** [comment]
@orhan-drsn Seems a good idea, Have you tested it ? if yes, I will make the update to implement that . thank you

**Comment by orhan-drsn** [comment]
@CharlesCCC yes it works for me.
It's like a real human would do it. After entering the passwort to push enter

**Comment by CharlesCCC** [comment]
@orhan-drsn Thank you for confirming that, I made the change and also tested it locally, seems working really well.

**Comment by converge** [comment]
@orhan-drsn great idea!

**Comment by timgrossmann** [comment]
@CharlesCCC @orhan-drsn Thank you very much for this, guys! This will remove a lot of problems in the future 🙌

**Pull request merged**